README.md

```diff
@@ -53,6 +56,8 @@ Select some text, click on the tooltip button in the inline-tools bar, and type
 | location          | `{location: string}` | You could choose the tooltip place(top, bottom, left, right). Default value is bottom
 | color | `{ color: Hex or string }`|You could choose the background text color wrapped by the tooltip, you could pass hexadecimal colors or string colors
 | underline | `{ underline: boolean }`  | You could add underline text decoration to the text wrapped by the tooltip
+| tooltipColor | `{ tooltipColor: string }`  | You could choose the tooltip color, you could pass hexadecimal colors or string colors
```

Comment:
@MarioRodriguezS not really sure but IMO, we should use more specific names like `backgroundColor`, `textColor` and `highlightColor`  What do you think?

Reply:
@juanhurtado10  comments solved. Thanks!